snap/snapcraft.yaml

```
@@ -31,6 +31,19 @@ apps:
       - mount-observe
       - hardware-observe
       - removable-media
+  shotwell-viewer:
+    command: launch.sh $SNAP/usr/bin/shotwell
+    extensions: [gnome]
+    desktop: usr/share/applications/org.gnome.Shotwell-Viewer.desktop
+    environment:
+      HOME: $SNAP_USER_COMMON
+    plugs:
+      - audio-playback
+      - home
+      - raw-usb
+      - mount-observe
+      - hardware-observe
+      - removable-media
```

Since the plugs are the same, I think that using a reference is better.

```suggestion
    plugs: *pluglist
```

---

You have to apply this one AND https://github.com/ubuntu/shotwell/pull/19/files#r2416861829